src/main/utils/goUtils.ts

```diff
@@ -45,9 +46,29 @@ export class GoUtils {
         dependenciesTreeNode: DependenciesTreeNode,
```

Comment:
Looks like this version is not in use anymore

Reply:
removed